classes/Http/Action/Admin/Talk/RateAction.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * Copyright (c) 2013-2017 OpenCFP
+ *
+ * For the full copyright and license information, please view
+ * the LICENSE file that was distributed with this source code.
+ *
+ * @see https://github.com/opencfp/opencfp
+ */
+
+namespace OpenCFP\Http\Action\Admin\Talk;
+
+use Illuminate\Container\Container;
+use Illuminate\Filesystem\Filesystem;
+use Illuminate\Translation;
+use Illuminate\Validation;
+use OpenCFP\Domain\Talk;
+use OpenCFP\Domain\ValidationException;
+use Symfony\Component\HttpFoundation;
+
+final class RateAction
+{
+    /**
+     * @var Talk\TalkHandler
+     */
+    private $talkHandler;
+
+    public function __construct(Talk\TalkHandler $talkHandler)
+    {
+        $this->talkHandler = $talkHandler;
+    }
+
+    public function __invoke(HttpFoundation\Request $request): HttpFoundation\Response
+    {
+        try {
+            $this->validate($request, [
+                'rating' => 'required|integer',
+            ]);
+        } catch (ValidationException $exception) {
+            return new HttpFoundation\Response();
```

Comment:
I know, this is the same behavior as in the controller, but I was thinking, we should maybe send a `400` status here instead of `200`. WDYT?

Reply:
Agreed.